source/help/moderation/index.md

```diff
@@ -22,16 +22,14 @@ arXiv, in its sole discretion, may decline to post works submitted to the platfo
 <span id="scholarly-standards"></span>
 #### Scholarly Standards
 
-Submissions to arXiv must meet appropriate scholarly standards. arXiv will decline submissions that do not meet these scholarly standards.  
+Submissions to arXiv must comply with appropriate standards of scholarly communication both in form (including appropriate and sufficiently carefully prepared sections, figures, tables, references, etc.), language (professional communication, see also [Code of Conduct](../../help/policies/code_of_conduct.md)), sufficiently neutral tone, and general scrupulousness and care of preparation.
```

Comment:
should the parentheticals in this be bumped to another sentence? This reads awkwardly.

Reply:
broke out into separate sentences